src/configure-nilrt-snac

```diff
@@ -140,6 +140,18 @@ remove_niauth() {
 	trap - EXIT
 }
 
+disable_wifi() {
+	log INFO Disabling WiFi...
+	cat <<EOT >/etc/modprobe.d/snac_blacklist.conf
+# Do not allow WiFi connections
+# We cannot use the blacklist keyword because they will still be loaded
+# when another module depends on them. This will prevent the modules from
+# being loaded along with any modules that depends on them.
+install cfg80211 /bin/true
+install mac80211 /bin/true
+EOT
```

Comment:
This will blacklist the modules from getting loaded, but what about removing them if they are already installed. Do we need some `rmmod` calls here?

Reply:
What does "install" really mean here? If it's referring to being loaded into memory, then will we expect the user to reboot after running the config script? Wouldn't it be never-loaded after that?

Reply:
"install" in the script means to load it into memory. They are still available on disk, I.e. installed on the system, but this prevents modprobe from using them.

The assumption here is that snac mode will not completely take effect until a reboot. If this is not the case, then yes we would need to call rmmod as part of this.

Reply:
The instructions direct the user to reboot today because we *have to*, but we shouldn't rely on that behavior in our design. It isn't normal to have to reboot a linux machine just because you insert or remove a kernel module (that's the point of a module.)

Reply:
Call rmmod